### ISSUE TITLE
No Proper Steps

### ISSUE THREAD

**Issue by ozamihir1990** [ethers-io/EthersWallet-ios#19]
after downloading the source getting an error 
![screen shot 2017-12-29 at 5 52 51 pm](https://user-images.githubusercontent.com/12641767/34437104-7fc46e2e-ecc1-11e7-9e84-b37758b349fc.png)
![screen shot 2017-12-29 at 5 54 45 pm](https://user-images.githubusercontent.com/12641767/34437105-8005770c-ecc1-11e7-9c2a-005d679b6631.png)

Please make an easy and understandable documentation for wallet implementation.

**Comment by ricmoo** [comment]
Yes, I need to add that to the documentation...

You need to download two repositories from this account, `EthersWallet-iOS` and `ethers.objc`. Place them in the same folder, and then try building the Xcode project in `EthersWallet-iOS`.

I will leave this ticket open to remind me to update the documentation.

Thanks!

**Comment by ozamihir1990** [comment]
Thanks.
 Which network provider you are using by default for creating or import wallet?

**Comment by ricmoo** [comment]
By default it creates a `FallbackProvider` with an `EtherscanProvider` and an `InfuraProvider`.

If one doesn't not respond or returns and error, the next takes over. As long as one of those services is up, it will work. I will be adding a EthersProvider in the non-distant future too; we are working hard on our own explorer. :)

**Comment by Badams451** [comment]
Any plans for support of ERC20 tokens?

**Comment by ricmoo** [comment]
@Badams451 Feel free to open a new issue to discuss this further, but yes. Currently a simple app is being built to support all ERC-20 tokens, but we are planning native ERC-20 and ERC-721 support. There is still backend work to do for history, which will also add support in the history for internal transactions.

**Comment by arunmitteam** [comment]
Any idea what I'm missing ? Thanks in advance!
EthersWallet-ios/EthersWallet/src/Utilities/Utilities.h:29:9: 'ethers/Promise.h' file not found

**Comment by ricmoo** [comment]
Do you have ethers.objc installed in the same directory as the EthersWallet-iOS is installed?

**Comment by arunmitteam** [comment]
Yes both the projects are in the same directory, ethers.objc build succeeded.

**Comment by arunmitteam** [comment]
I got it working now, Thanks for your quick response.
Let me know if I could help in any of the issues/enhancements.

**Comment by konohateo** [comment]
Hi, I've still unable to build it, can anyone provide the instruction?
<img width="1393" alt="screen shot 2018-02-19 at 3 29 03 pm" src="https://user-images.githubusercontent.com/19849878/36366681-2e491468-158a-11e8-9c02-5427f76e283b.png">
